tests/Microsoft.Identity.Test.Android.UIAutomation/androidTests.cs

```diff
@@ -251,7 +278,9 @@ public void AcquireTokenADFSV2019InteractiveFederatedTest()
         [Test]
         public void AcquireTokenADFSV3InteractiveFederatedTest()
         {
-            _mobileTestHelper.AcquireTokenInteractiveTestHelper(_xamarinController, LabUserHelper.GetAdfsUserAsync(FederationProvider.AdfsV3).GetAwaiter().GetResult());
+            _mobileTestHelper.AcquireTokenInteractiveTestHelper(
```

Comment:
Similar to my previous comment - do we expect tests hitting ADFSV3, ADFSv2 etc. to catch any issues? I'd say no because: 

- we have a mobile test hitting AAD, which tests the mobile web UI
- we have desktop tests hitting all the ADFS users

The only thing we would ever catch is EVO sending custom UI for mobiles and this UI being broken. @henrik-me - is this worth it?